.coveragerc

```diff
@@ -238,6 +238,7 @@ omit =
     homeassistant/components/notify/xmpp.py
     homeassistant/components/nuimo_controller.py
     homeassistant/components/openalpr.py
+    homeassistant/components/remote/harmony.py
```

Comment:
Please don't change this files permissions. Makes me wonder if all the new files have the correct 644 permission ?